### ISSUE TITLE
publicPath set with http will not work

### ISSUE THREAD

**Issue by muei** [quasarframework/quasar#4460]
If set *publicPath* with `http` prefix, the HTML tag `base` use it, and the `VUE_ROUTER_BASE` will use it too, so this will not work. And the trunk files will not request as cdn host. If deployed online, will throw error:
```
Uncaught DOMException: Failed to execute 'replaceState' on 'History': A history state object with URL 'https://cdn.XXX.com/e13523fe-b9a5-4306-8967-6610e98ca18d' cannot be created in a document with origin 'https://XXX.com' and URL ...
```

**Comment by rstoenescu** [comment]
I really must ask before digging deeper: why would you need a publicPath which includes the protocol and domain?

**Comment by rstoenescu** [comment]
Fix will be available in final 1.0.0.
Still, would like to know why would you need a publicPath which includes the protocol and domain? Maybe I am missing something :)

**Comment by muei** [comment]
For unified manage the js, css, fonts... as cdn
